app/serializers/current_user_serializer.rb

```diff
@@ -76,7 +76,6 @@ class CurrentUserSerializer < BasicUserSerializer
              :pending_posts_count,
              :status,
              :sidebar_category_ids,
-             :sidebar_tags,
```

Comment:
hmm I think we still need this?

Reply:
I'm actually confused now because if we remove this, how are we shipping the attribute down to the client?

Reply:
yeah, it was duplicated. We have it here https://github.com/discourse/discourse/blob/main/app/serializers/concerns/user_sidebar_tags_mixin.rb

Reply:
ahhh ok xD Good catch :+1: